scripts/common.psm1

```diff
@@ -16,23 +16,31 @@ function Invoke-Block([scriptblock]$cmd) {
     }
 }
 
-function Get-Submodules([string]$ModuleDirectory)
-{
-    Invoke-Block { & git submodule update --init }
+function Get-Submodules {
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$RepoRoot
+    )
 
-    $gitModules = Join-Path $RepoRoot ".gitmodules"
+    if ((Get-ChildItem "$RepoRoot/modules/*" -Directory | Measure-Object) -eq 0) {
+        Write-Verbose "git submodule update --init"
+        Invoke-Block { & git submodule update --init }
```

Comment:
Nit: might be nice if the contents of this block and Write-Verbose were driven by the same variable to prevent drift.